src/configdialog.cpp

```diff
@@ -244,7 +244,7 @@ void ShortcutDelegate::setModelData(QWidget *editor, QAbstractItemModel *model,
 				if (UtilsUi::txsConfirmWarning(QString(ConfigDialog::tr("The shortcut <%1> is already assigned to the command:")).arg(value) + "\n" + duplicate + "\n\n" + ConfigDialog::tr("Do you wish to remove the old assignment and bind the shortcut to the new command?"))) {
 					//model->setData(mil[0],"",Qt::DisplayRole);
 					foreach (QTreeWidgetItem *twi, li) {
-						if (twi and twi->text(2) == value) twi->setText(2, "");
+                        if (twi && twi->text(2) == value) twi->setText(2, "");
```

Comment:
Can you please check the code formatting settings of your editor? We're using tabs for indentation and your edits replace them by 4 spaces.

Reply:
I'm sorry about that! Fix: #429